frontend/src/components/pages/SpaceListingPage.tsx

```diff
@@ -1,10 +1,43 @@
-import { CreateSpaceListingArea } from "../compound/CreateSpaceListingArea"
+import { CreateSpaceListingArea } from "../compound/CreateSpaceListingArea";
+import SpaceListingSection from "./SpaceListingsSection";
+import SpaceListingCard from "../compound/SpaceListing";
+import UserListing from "../compound/UserListing";
 
 export const SpaceListingPage = () => {
-    return (
-        <>
-            <CreateSpaceListingArea />
+  const space_listing = {
```

Comment:
FYI make sure these aren't added back - we are deleting these from https://github.com/fractal-bootcamp/directoryNY/pull/35